<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureManager</class>
 <widget class="QWidget" name="TextureManager">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="textureManagerLayout" columnstretch="2,3">
   <item row="1" column="0" rowspan="2">
    <layout class="QHBoxLayout" name="buttons">
     <item>
      <widget class="QToolButton" name="createTextureButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>New</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteTextureButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPropertyEditorWidget" name="textureProperties"/>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="QScrollArea" name="imagePreviewContainer">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="imagePreviewContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>295</width>
        <height>429</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="imageLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QListWidget" name="textureList"/>
   </item>
  </layout>
  <action name="actionFrom_Image">
   <property name="text">
    <string>From Image</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionFrom_Gradient">
   <property name="text">
    <string>From Gradient</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPropertyEditorWidget</class>
   <extends>QTreeView</extends>
   <header>QPropertyEditor/QPropertyEditorWidget.h</header>
   <slots>
    <slot>addObject(QObject*)</slot>
    <slot>setObject(QObject*)</slot>
    <slot>updateObject(QObject*)</slot>
    <slot>removeObject(QObject*)</slot>
    <slot>setObject(QListWidgetItem*)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>textureList</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>textureProperties</receiver>
   <slot>setObject(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createTextureButton</sender>
   <signal>clicked()</signal>
   <receiver>createTextureButton</receiver>
   <slot>showMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
